src/components/DivElement.js

```diff
@@ -1,5 +1,13 @@
 // Import HTMLElement here
+import HTMLElements from './HTMLElement';
 
 // Define class here
-
+class DivElement extends HTMLElements {
+  constructor(content) {
+    super(content);
```

Comment:
```suggestion
    super('div', content);
```
Then you don't need lines 8 or 9